jira/client.py

```diff
@@ -2376,14 +2377,14 @@ def reindex(self, force=False, background=True):
 
         r = self._session.get(url, headers=self._options['headers'])
         if r.status_code == 503:
-            # logging.warning("JIRA returned 503, this could mean that a full reindex is in progress.")
+            # log.warning("JIRA returned 503, this could mean that a full reindex is in progress.")
```

Comment:
@vkrizan Ok, rename it to `log` and keep the constructor signature but document that the parameter is deprecated and ignored. Once we have this we will merge this PR and move to fix others. Thanks.